packages/manager/modules/billing/package.json

```
@@ -11,12 +11,21 @@
   "license": "BSD-3-Clause",
   "author": "OVH SAS",
   "main": "./src/index.js",
+  "dependencies": {
+    "@ovh-ux/manager-config": "^2.0.0",
```

```suggestion
    "@ovh-ux/manager-config": "^2.0.0",
```
could these be added in peerDependencies?

---

@radireddy it's a non angular dependency